scripts/osm_pbf_power_data_extractor.py

```diff
@@ -429,7 +429,8 @@ def output_csv_geojson(output_files, country_code, df_all_feature, columns_featu
     # Generate Files
 
     if df_all_feature.empty:
-        _logger.warning(f"All feature data frame empty for {feature}")
+        _logger.warning(f"Store empty Dataframe for {feature}.")
+        gdf_feature = []
```

Comment:
Ups this is also a small fix. Countries with empty data frames were causing an error. This code addition resolves that.